settings.yaml.example

```diff
@@ -0,0 +1,156 @@
+
```

Comment:
Please include link to the original documentation for reference https://microsoft.github.io/graphrag/config/json_yaml/

Reply:
done